### PR TITLE
feat(kit): add alias `tui-textarea` for `tui-text-area`

### PR DESCRIPTION
**Pull request by splincode**
## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring
- [ ] Build or CI related changes
- [ ] Tests related changes
- [ ] Documentation content changes

### CODE REVIEW

File: projects/demo/src/modules/app/pages.ts [hunk 1]
```diff
@@ -514,9 +514,9 @@ export const pages: TuiDocPages = [
             },
             {
                 section: `Components`,
-                title: `TextArea`,
-                keywords: `поле, инпут, форма, ввод, text-area, area`,
-                route: `/components/text-area`,
+                title: `Textarea`,
+                keywords: `поле, инпут, форма, ввод, textarea, text-area, area`,
```
Comment (waterplea) [inline projects/demo/src/modules/app/pages.ts:518]: Unnecessary `textarea` since it matches the title :)

**Review by waterplea — COMMENTED** [review]
Can we also rename module/component and deprecate old capitalization? A migration script would also be necessary.

**Comment by splincode** [comment]
@waterplea hello, please review me

**Review by waterplea — APPROVED** [review]

File: projects/kit/components/textarea/textarea.template.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
 >
     <div
         *ngIf="hasCounter"
-        automation-id="tui-text-area__counter"
+        automation-id="tui-textarea__counter"
```
Comment (waterplea) [inline projects/kit/components/textarea/textarea.template.html:15]: Can we exempt `automation-id` from migration and leave them as they are?

Comment (splincode) [reply projects/kit/components/textarea/textarea.template.html:15]: i'm not sure if it's easy

File: projects/kit/components/textarea/textarea.directive.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import {Directive} from '@angular/core';
+import {AbstractTuiTextfieldHost, tuiAsTextfieldHost} from '@taiga-ui/core';
+
+import {TuiTextareaComponent} from './textarea.component';
+
+@Directive({
+    selector: 'tui-textarea, tui-text-area',
```
Comment (vladimirpotekhin) [inline projects/kit/components/textarea/textarea.directive.ts:7]: nit: maybe add TODO to delete `tui-text-area` in 4.0 ?

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**